### ISSUE TITLE
3.2.4 Consistent Identification - understanding should include "accessible name" concept?

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#300]
the current understanding doc for 3.2.4 https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html mentions:

> This consistency extends to the text alternatives. If icons or other non-text items have the same functionality, then their text alternatives should be consistent as well.

I think this should actually go further and include generally the *accessible name*. i.e. the consistency should also apply if the components make use of, say, `aria-label`, `aria-labelledby` or any other way in which the name (as exposed by AT) is calculated.

**Comment by alastc** [comment]
Just going through old issues... isn't the accessible name the output of the text-alterantive?

It doesn't say alt-text, it says "text alternative", trying to be tech-neutral.

I see your point, I just think it would apply at the technique level rather than the higher-level understanding doc.

Overall, if you (or anyone) wants to make a PR we'll put it forward, but it's relatively minor and if no-one puts in that effort, it would be best to close.
